app/views/shared/_money_input.html.slim

```diff
@@ -1,7 +1,8 @@
 = form.govuk_text_field field,
                         width:,
-                        label: { text: label_text },
+                        label: { text: label_text, size: "m" },
```

Comment:
Hang on, this is going to change the size of the label for money inputs on _every_ screen, which means we'll end up with some fields on screens looking distinctly odd, e.g.:
<img width="511" alt="Screenshot 2022-12-13 at 15 23 13" src="https://user-images.githubusercontent.com/113888736/207373594-83b727bd-a763-4cd5-99cd-3edd1f143ea2.png">
(Until now the type and amount labels have been the same size)

Do we want to pass in size as an optional parameter to this partial, so that it defaults to its default ('s'?), and can be overridden when used on the employment screen?

Reply:
Done that.